scripts/benchmark/checks/simple_value_transfers.py

```diff
@@ -0,0 +1,121 @@
+import logging
+
+from typing import (
+    NamedTuple
+)
+
+from eth_typing import (
+    Address
+)
+from evm.chains.base import (
+    Chain
+)
+from evm.rlp.blocks import (
+    BaseBlock,
+)
+
+from .base_benchmark import (
+    BaseBenchmark
+)
+from utils.chain_plumbing import (
+    FUNDED_ADDRESS,
+    FUNDED_ADDRESS_PRIVATE_KEY,
+    get_all_chains,
+    NON_EXISTING_ADDRESS,
+    SECOND_EXISTING_ADDRESS,
+)
+from utils.reporting import (
+    DefaultStat
+)
+from utils.shellart import (
+    bold_yellow
+)
+from utils.tx import (
+    new_transaction
+)
+
+
+class SimpleValueTransferBenchmarkConfig(NamedTuple):
+    to_address: Address
+    greeter_info: str
+
+
+TO_EXISTING_ADDRESS_CONFIG = SimpleValueTransferBenchmarkConfig(
+    to_address=SECOND_EXISTING_ADDRESS,
+    greeter_info='Sending to existing address\n'
+)
+
+
+TO_NON_EXISTING_ADDRESS_CONFIG = SimpleValueTransferBenchmarkConfig(
+    to_address=NON_EXISTING_ADDRESS,
```

Comment:
This won't actually work.  You need to generate a new address for each transaction to benchmark the thing we're actually trying to test, which is the additional overhead we incur for transactions which create new accounts in the state database.  If we re-use the same address then only the first transaction triggers this and the rest are just simple value transfers.

Reply:
Oops, of course! 🙈